doozer/doozerlib/cli/images.py

```diff
@@ -376,6 +376,105 @@ def dgr_rebase(image_meta, terminate_event):
         raise DoozerFatalError('One or more images failed. See state.yaml')
 
 
+@cli.command("k:images:rebase", short_help="Refresh a group's distgit content from source content.")
+@click.option("--version", metavar='VERSION', default=None, callback=validate_semver_major_minor_patch,
+              help="Version string to populate in Dockerfiles. \"auto\" gets version from atomic-openshift RPM")
+@click.option("--release", metavar='RELEASE', default=None, help="Release string to populate in Dockerfiles.")
+@click.option("--embargoed", is_flag=True, help="Add .p1 to the release string for all images, which indicates those images have embargoed fixes")
+@click.option("--repo-type", metavar="REPO_TYPE", envvar="OIT_IMAGES_REPO_TYPE",
+              default="unsigned",
+              help="Repo group type to use for version autodetection scan (e.g. signed, unsigned).")
+@click.option("--force-yum-updates", is_flag=True, default=False,
+              help="Inject \"yum update -y\" in the final stage of an image build. This ensures the component image will be able to override RPMs it is inheriting from its parent image using RPMs in the rebuild plashet.")
+@option_commit_message
+@option_push
+@pass_runtime
+def k_images_rebase(runtime: Runtime, version: Optional[str], release: Optional[str], embargoed: bool, repo_type: str, force_yum_updates: bool, message: str, push: bool):
```

Comment:
I suggest a metadata marker in the component that indicates that is should be part of konflux rebases / builds. This will reduce the load on our systems and make iteration much faster during early development. We only need a handful of images rebasing  / building in Konflux to prove out the theory. Something very simple like:
```
konflux:
  enabled: true
```

Reply:
added in https://github.com/openshift-eng/art-tools/pull/543/commits/20ff4db345bfc9f5275e44da5753ae9d7cec7e5c